tests/v2/test_peer_manager.nim

```diff
@@ -76,10 +76,11 @@ procSuite "Peer Manager":
     await allFutures(nodes.mapIt(it.start()))
     await allFutures(nodes.mapIt(it.mountRelay()))
 
-    let nonExistentPeer = parseRemotePeerInfo("/ip4/0.0.0.0/tcp/1000/p2p/16Uiu2HAmL5okWopX7NqZWBUKVqW8iUxCEmd5GMHLVPwCgzYzQv3e")
+    let nonExistentPeer = parsePeerInfo("/ip4/0.0.0.0/tcp/1000/p2p/16Uiu2HAmL5okWopX7NqZWBUKVqW8iUxCEmd5GMHLVPwCgzYzQv3e")
+    require nonExistentPeer.isOk()
 
     # Dial non-existent peer from node1
-    let conn1 = await nodes[0].peerManager.dialPeer(nonExistentPeer, WakuFilterCodec)
+    let conn1 = await nodes[0].peerManager.dialPeer(nonExistentPeer.value, WakuFilterCodec)
```

Comment:
mind doing the following?

```
let nonExistentPeerRes = parsePeerInfo("/ip4/0.0.0.0/tcp/1000/p2p/16Uiu2HAmL5okWopX7NqZWBUKVqW8iUxCEmd5GMHLVPwCgzYzQv3e")

... checks

let nonExistentPeer = nonExistentPeerRes.value
```


Like this you dont need to have the boilerplate `.value` `.value`, so less lines of code to modify, less diff and perhaps clearer?

We kind of have this pattern in other places. `xxxRes`, then checks and then `xxx = xxxRes.value` (or get)

Reply:
I will do that thanks !